src/211/main/kotlin/org/rust/ide/formatter/RustfmtExternalFormatProcessor.kt

```diff
@@ -66,17 +65,12 @@ abstract class RustfmtExternalFormatProcessorBase : ExternalFormatProcessor {
 
     override fun getId(): String = "rustfmt"
 
-    override fun activeForFile(source: PsiFile): Boolean {
-        return (ApplicationInfo.getInstance().build < BUILD_212 || !reformatInProcess.get()) && isActiveForFile(source)
-    }
+    override fun activeForFile(source: PsiFile): Boolean = isActiveForFile(source)
 
     // Never used by the platform?
     override fun indent(source: PsiFile, lineStartOffset: Int): String? = null
 
     companion object {
-        // BACKCOMPAT: 2021.1
-        private val BUILD_212: BuildNumber = BuildNumber.fromString("212")!!
-        // TODO: implement `FormattingService` instead
         private val reformatInProcess: ThreadLocal<Boolean> = ThreadLocal.withInitial { false }
```

Comment:
I expect that this hack with thread local property will be removed in this PR

Reply:
Fixed.